modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesRuntimeProvisioningService.java

```diff
@@ -593,7 +598,7 @@ public void deleteCustomDomain(String domain) throws RuntimeProvisioningExceptio
 
             httpclient = KubernetesProvisioningUtils.getHttpClientForKubernetes();
             uri = new URI(KubernetesPovisioningConstants.KUB_MASTER_URL + KubernetesPovisioningConstants.INGRESS_API_NAMESPACE_RESOURCE_PATH
-                          + KubernetesProvisioningUtils.getNameSpace(applicationContext).getMetadata().getNamespace()
+                          + namespace.getMetadata().getNamespace()
                           + KubernetesPovisioningConstants.INGRESS_API_RESOURCE_PATH_SUFFIX);
             HttpDelete httpDelete = (HttpDelete) KubernetesProvisioningUtils.getHttpMethodForKubernetes(HttpDelete.METHOD_NAME, uri);
             httpDelete.addHeader(HttpHeaders.CONTENT_TYPE, KubernetesPovisioningConstants.MIME_TYPE_JSON);
```

Comment:
This code is no longer use in AF, because we moved to Fabric8 client for ingress creation.